### PR TITLE
remove poll transaction

### PR DESCRIPTION
**Pull request by yash431garg**
Poll transaction method is returning confirmed event the tx fails if user have insufficient bonk and have a unused nft.

Screenshot 2024-12-20 at 7 23 24 PM

### CODE REVIEW

**Comment by yash431garg** [comment]
@samott This is a urgent fix. You can check the whole issue in this slack thread https://gator-labs.slack.com/archives/C06054PPNSY/p1734701225694339

File: app/actions/chompResult.ts [hunk 1]
```diff
@@ -447,55 +440,56 @@ async function hasBonkBurnedCorrectly(
...
+//           },
+//         });
+
+//         await Sentry.flush(SENTRY_FLUSH_WAIT);
+//         clearInterval(intervalId);
+//         resolve(false); // Return false if an error occurs
+//       }
+//     }, interval);
+//   });
+// }
```
Comment (samott) [inline app/actions/chompResult.ts:493]: Is this code going to come back later?

**Comment by neel-ds** [comment]
It correctly verifies the txn confirmation status.

File: app/actions/chompResult.ts (hunk 1 above)
Comment (yash431garg) [reply app/actions/chompResult.ts:493]: No,

File: app/actions/chompResult.ts [hunk 2]
```diff
@@ -447,55 +440,56 @@ async function hasBonkBurnedCorrectly(
  *
  * @return status boolean
  */
-async function pollTransactionConfirmation(
```
Comment (neel-ds) [inline app/actions/chompResult.ts:450]: If this function is not required then can you remove it?

**Review by neel-ds — APPROVED** [review]

**Review by samott — APPROVED** [review]

**Pull request merged**